strategy/json/sql/postgres/single_stream_strategy.go

```diff
@@ -90,12 +90,47 @@ func (s *SingleStreamStrategy) PrepareData(messages []goengine.Message) ([]inter
 			payloadType,
 			payloadData,
 			meta,
+			msgMetadata.Value("_aggregate_type"),
```

Comment:
Didn't we remove these from the metadata?

Reply:
We removed them, but they are being used to pass the values along the message's lifecycle. The query used to read from the event streams is still
```
SELECT "no", "event_id", "event_name", "payload", "metadata", "created_at" FROM events_box_delivery WHERE no >= $1 AND aggregate_type = $2 AND aggregate_id = $3 ORDER BY no
```, 

But we do not query these new fields. As opposed to the previous PSQL JSON query syntax.